sdks/python/apache_beam/utils/profiler.py

```diff
@@ -134,10 +139,8 @@ def stop(self):
 
   def report_once(self):
     if not self._hpy:
-      logging.warning('guppy is not installed; MemoryReporter not available.')
       return
     report_start_time = time.time()
     heap_profile = self._hpy().heap()
-    logging.info('*** MemoryReport Heap:\n %s', heap_profile)
-    logging.info('*** MemoryReport took %.1f sec.',
-                 time.time() - report_start_time)
+    logging.info('*** MemoryReport Heap:\n %s\n MemoryReport took %.1f sec',
```

Comment:
It goes more than thousand for complicated pipelines. I would dump to disk instead, as cpu profiler does. But I'd like to do it in a separate PR, when it's plugged into DirectRunner using pipeline options. WDYT?